src/main/java/io/workflow/bpmnsimulator/fieldvalidator/FieldValidator.java

```diff
@@ -0,0 +1,11 @@
+package io.workflow.bpmnsimulator.fieldvalidator;
+
+import io.workflow.bpmnsimulator.model.Step;
+import org.camunda.bpm.engine.task.Task;
+
+import javax.annotation.Nonnull;
+
+public interface FieldValidator {
```

Comment:
can be rename to Validator

Reply:
Sure. It's done here:
https://github.com/mohsen-ebrahimi/bpmn-simulator/pull/3